src/components/EventForm.js

```diff
@@ -266,3 +264,15 @@ export default class EventForm extends Component {
     );
   }
 }
+
+const mapStateToProps = state => ({
+  user: state.user
+});
+
+const mapDispatchToProps = dispatch => ({
+});
```

Comment:
Looks like we don't need to include mapDispatchToProps if we're not going to use it?

Reply:
You're right, taking this out once I make sure it's not needed.